VRDR/DeathRecord.cs

```diff
@@ -6348,22 +6348,57 @@ public string DeathLocationTypeHelper
                 {
                     CreateAgeAtDeathObs(); // Create it
                 }
-                string extractedValue = GetValue(value, "value");
+                string extractedValue = GetValue(value, "units");
                 // If the value or unit is null, put out a data absent reason
                 if ( !String.IsNullOrWhiteSpace(extractedValue) ){  // if there is a value for age, set it
                     Quantity quantity = (Quantity)AgeAtDeathObs.Value;
                     quantity.Value = Convert.ToDecimal(extractedValue);
                     AgeAtDeathObs.Value = quantity;
                     AgeAtDeathDataAbsentBoolean = false; // if age is present, cancel the data absent reason
                 }
-                if( (!String.IsNullOrWhiteSpace(GetValue(value, "unit"))) ){ // if there is a value for unit, set it
+                extractedValue = GetValue(value, "type");
+                if (!String.IsNullOrWhiteSpace(extractedValue))
+                { // if there is a value for unit, set it
                     Quantity quantity = (Quantity)AgeAtDeathObs.Value;
-                    quantity.Unit = GetValue(value, "unit");
+                    quantity.Unit = extractedValue;
                     AgeAtDeathObs.Value = quantity;
+                    AgeAtDeathDataAbsentBoolean = false; // if age is present, cancel the data absent reason
                 }
               }
         }
 
+        [Property("Age At Death Data Absent Reason (Code)", Property.Types.Dictionary, "Death Investigation", "Age At Death Edit Bypass Flag.", true, "http://build.fhir.org/ig/HL7/vrdr/StructureDefinition-vrdr-decedent-age.html", true, 2)]
+        [PropertyParam("code", "The code used to describe this concept.")]
+        [PropertyParam("system", "The relevant code system.")]
+        [PropertyParam("display", "The human readable version of this code.")]
+        [PropertyParam("text", "Additional descriptive text.")]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='39016-1').extension.where(url='http://hl7.org/fhir/us/vrdr/ValueSet/vrdr-edit-bypass-01-vs')", "")]
+        public Dictionary<string, string> AgeAtDeathEditBypassFlag
```

Comment:
The extension URL should be ExtensionURL.BypassEditFlag.   The URL you have is the URL of the value set used by this instance of the extension.